test/test-code/2-game-slug-release.js

```diff
@@ -0,0 +1,221 @@
+import {
+  gameReleasesURL,
+  browser,
+  createUserGroupGameRelease,
+  login,
+  sleep,
+  Release,
+  isLoginPage
+} from '../helpers';
+import { expect } from 'chai';
+import { until, By, error, WebElement } from 'selenium-webdriver';
+/**
+ *
+ * @param {0 | 1 | 2} permission
+ * @param {0 | 1 | 2} privilege
+ * @param {"dev" | "qa" | "stage" | "prod"} gameStatus
+ */
+export const basic = async (permission, privilege, gameStatus) => {
+  const { user, game } = await createUserGroupGameRelease({
+    permission,
+    privilege,
+    gameStatus
+  });
+  const url = gameReleasesURL(game);
+  await login(user);
+
+  await browser.get(url);
+
+  const err = await browser
+    .wait(until.elementsLocated(By.css('[data-target="#addRelease"]')), 500)
+    .catch(err => err);
+
+  expect(err).to.not.be.instanceOf(error.TimeoutError);
+
+  const text = await browser.findElement(By.className('panel-title')).getText();
+
+  expect(text).to.equal('Releases');
+  return game;
+};
+
+export const publicUserTest = async () => {
+  const { game } = await createUserGroupGameRelease();
+  const url = gameReleasesURL(game);
+  await browser.get(url);
+
+  await isLoginPage();
+};
+
+/**
+ *
+ * @param {boolean} pass
+ */
+export const changeTest = async pass => {
```

Comment:
You could, but then what if the data in the db is getting updated, but the user does not see that updated data, then it would still be classified as a bug. I would consider checking the text to be more reliable